### DISCUSSION TITLE
Accessing temp directories when testing with ExUnit

### DISCUSSION THREAD

**Discussion by ndrean** [dwyl/image-classifier#100] [General]
[<img width="994" alt="Screenshot 2024-02-01 at 10 49 55" src="https://github.com/dwyl/image-classifier/assets/6793008/061490cf-9671-401d-bd81-628e2aeffb15">](<https://www.elixirstreams.com/tips/easy-concurrent-temp-dirs-with-exunit?utm_source=elixir-merge>)

Doc: <https://hexdocs.pm/ex_unit/main/ExUnit.Case.html#module-tmp-dir>

**Comment by LuchoTurtle** [comment]
Thank you for sharing this, I've already used this little trick and it saved me a lot of headaches, especially when testing edge cases with different file types!
